tests/Tests/ORM/Proxy/ProxyFactoryTest.php

```diff
@@ -209,6 +211,18 @@ public function testProxyClonesParentFields(): void
         self::assertSame(1000, $cloned->getSalary(), 'Expect properties on the CompanyEmployee class to be cloned');
         self::assertSame('Bob', $cloned->getName(), 'Expect properties on the CompanyPerson class to be cloned');
     }
+
+    #[RequiresPhp('8.4')]
+    #[DoesNotPerformAssertions]
+    public function testProxyFactoryAcceptsNullProxyArgsWhenNativeLazyObjectsAreEnabled(): void
```

Comment:
I'd rather see a proxy created by the ProxyFactory than having this test without assertions.